packages/@tinacms/graphql/src/mdx/parse.ts

```diff
@@ -26,7 +26,7 @@ import type { RichTypeInner } from '../types'
 import { isNull } from 'lodash'
 
 export const parseMDX = (value: string, field: RichTypeInner) => {
-  const tree = unified().use(markdown).use(mdx).parse(value)
+  const tree = unified().use(markdown, { commonmark: true }).use(mdx).parse(value)
```

Comment:
I was getting a type error here in the editor. I am not 100% sure if this is doing what you want.